pcstac/pcstac/middleware.py

```diff
@@ -0,0 +1,63 @@
+from http.client import HTTP_PORT, HTTPS_PORT
+from typing import List
+
+from stac_fastapi.api.middleware import ProxyHeaderMiddleware
+from starlette.types import Receive, Scope, Send
+
+
+class ProxyHeaderHostMiddleware(ProxyHeaderMiddleware):
```

Comment:
I'm not sure this is required. This was specifically to work around some headers that were introduced/modified due to a reverse proxy within spatio + dapr. I think the default middlewares from stac-fastapi would still be suitable here.